package.json

```diff
@@ -39,7 +39,7 @@
   "devDependencies": {
     "@nomiclabs/buidler": "1.1.2",
     "@nomiclabs/buidler-web3": "1.1.2",
-    "@openzeppelin/test-helpers": "0.5.4",
+    "@openzeppelin/test-helpers": "^0.5.5",
```

Comment:
please set as fixed version.
remove ^

So we have control when we change our work environment.

Reply:
change as request